app/src/main/java/com/huji/foodtricks/buddies/ViewSingleEvent.java

```diff
@@ -106,4 +116,13 @@ public boolean onTouch(View v, MotionEvent event) {
             }
         });
     }
+
+    public void send_event_click(View view) {
```

Comment:
send to who? what does this do?

Reply:
In case the event status is 'pending' and the owner wants to move it to 'upcoming'